lib/NormalModule.js

```diff
@@ -136,17 +135,32 @@ class NormalModule extends Module {
 	}
 
 	createLoaderContext(resolver, options, compilation, fs) {
+		const addErrorFrom = (error) => {
+			if(!error.from) {
```

Comment:
When the error already contains a `from` property, prepend our own info.

Reply:
@sokra Thanks for remind this. But, btw, if not use a property in Error, how should we know the Error's loader name that threw from `runLoaders`? these were many of loaders.

Reply:
I mean in here: https://github.com/webpack/webpack/blob/master/lib/NormalModule.js#L222
It should have a way to know which loader it is. I was using `.from` property passed from `oad-runner`. What's your opinion?

Reply:
I just figure out that it also can use `loaderContext.loaderIndex` to get `currentLoader` even it has stoped by error occurred, haha.